Azure/azure-sdk-for-python
# [ML][Pipelines] Enable skip tests during GitHub migration

# Description

Enable skip tests during GitHub migration:

- enable 3 tests that requires GPU resource
- make component download tests only run in live test
- enable test that verifies service-side behavior in live test (we have tests that ensure client-side behaviors, e.g. code asset id)

For skipped AutoML DSL tests, may enable them in separate PR as it may need include AutoML devs to review.

# All SDK Contribution checklist:
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
